### PR TITLE
Next

### PR DESCRIPTION
**Pull request by zouxuoz**

### CODE REVIEW

File: src/components/Button/Button.js [hunk 1]
```diff
@@ -25,21 +23,19 @@ export type ButtonProps = {
   /** possible button types */
   type?: 'submit' | 'button' | 'reset',
   /** possible button colors */
-  color?: 'primary' | 'secondary' | 'neutral' | 'danger',
+  color?: 'primary' | 'neutral' | 'danger',
   /** the type of button */
-  variant?: 'outlined' | 'raised' | 'ghost',
```
Comment (ej9x) [inline src/components/Button/Button.js]: Why did you remove `secondary` color and `ghost` variant?

File: src/components/Checkbox/Checkbox.test.js [hunk 2]
```diff
@@ -3,15 +3,15 @@ import React from 'react';
...
+  it.skip('should call onChange callback', () => {
     const onChange = jest.fn();
     const wrapper = mount(<Checkbox onChange={ onChange } checked />);
 
     wrapper.find('input').simulate('change', { target: { checked: false }});
     expect(onChange.mock.calls[0][0]).toBe(false);
   });
 
-  it('should not call onChange callback with disabled prop', () => {
+  it.skip('should not call onChange callback with disabled prop', () => {
```
Comment (ej9x) [inline src/components/Checkbox/Checkbox.test.js]: For what?

File: src/components/TextArea/TextArea.js [hunk 3]
```diff
@@ -19,11 +19,11 @@ const theme = createComponentTheme(name, ({ COLORS, SIZES }: *) => ({
   root: {
     border: `1px solid ${COLORS.PRIMARY_BORDER_COLOR}`,
     borderRadius: SIZES.MAIN_BORDER_RADIUS,
-    fontSize: SIZES.MAIN_FONT_SIZE,
+    fontSize: SIZES.BODY_TEXT,
     color: COLORS.DARK_GRAY1,
     fontWeight: 400,
     lineHeight: 2,
-    padding: '1rem',
+    padding: '10px',
```
Comment (ej9x) [inline src/components/TextArea/TextArea.js]: `10px`?

File: src/components/Button/Button.js (hunk 1 above)
Comment (zouxuoz) [reply src/components/Button/Button.js]: @ej9x we never use it  if out DSM provide other button styles later - we can add it

File: src/components/Avatar/Avatar.js [hunk 4]
```diff
@@ -17,24 +17,24 @@ const theme = createComponentTheme(name, {
...
-        width: '3rem',
-        height: '3rem',
+        width: '30px',
+        height: '30px',
       },
       md: {
-        width: '5rem',
-        height: '5rem',
+        width: '50px',
+        height: '50px',
```
Comment (ej9x) [inline src/components/Avatar/Avatar.js]: `50px`

File: src/components/Card/CardSection.js [hunk 5]
```diff
@@ -24,7 +24,7 @@ const cardSectionTheme = createComponentTheme(name, ({ COLORS }: *) => ({
     hoverable: {
       cursor: 'pointer',
       '&:hover': {
-        boxShadow: '0 0.2rem 1rem 0 rgba(208, 215, 221, 0.6)',
+        boxShadow: '0 2px 10px 0 rgba(208, 215, 221, 0.6)',
```
Comment (ej9x) [inline src/components/Card/CardSection.js]: `10px`

File: src/components/Input/Input.theme.js [hunk 6]
```diff
@@ -67,24 +67,24 @@ const iconsStyles = {
   justifyContent: 'center',
   alignItems: 'center',
   height: '100%',
-  width: '3rem',
+  width: '30px',
```
Comment (ej9x) [inline src/components/Input/Input.theme.js]: `30px`

File: src/components/Loader/Loader.js [hunk 7]
```diff
@@ -13,16 +13,16 @@ const theme = createComponentTheme(name, {
   modifiers: {
     size: {
       sm: {
-        width: '4rem',
-        height: '4rem',
+        width: '30px',
+        height: '30px',
```
Comment (ej9x) [inline src/components/Loader/Loader.js]: `30px`

File: src/components/Avatar/Avatar.js [hunk 8]
```diff
@@ -17,24 +17,24 @@ const theme = createComponentTheme(name, {
...
-        width: '2rem',
-        height: '2rem',
+        width: '20px',
+        height: '20px',
       },
       sm: {
-        width: '3rem',
-        height: '3rem',
+        width: '30px',
+        height: '30px',
```
Comment (ej9x) [inline src/components/Avatar/Avatar.js]: `30px`

File: src/components/Code/Code.theme.js [hunk 9]
```diff
@@ -55,29 +55,29 @@ const CodePlateTag = createStyledTag(`${name}Plate`, {
...
+  padding: '20px 0 40px',
   borderTopLeftRadius: props.theme.SIZES.MAIN_BORDER_RADIUS,
   borderBottomLeftRadius: props.theme.SIZES.MAIN_BORDER_RADIUS,
 }));
 
 const CodeNumberWrapperTag = createStyledTag(`${name}NumberWrapper`, (props) => ({
   height: props.height ? `${props.height}px` : 'auto',
   display: 'block',
-  width: '3rem',
+  width: '30px',
```
Comment (ej9x) [inline src/components/Code/Code.theme.js]: `30px`

File: src/components/Code/Code.theme.js [hunk 10]
```diff
@@ -55,29 +55,29 @@ const CodePlateTag = createStyledTag(`${name}Plate`, {
...
   flexShrink: 0,
   position: 'relative',
 }));
 
 const CodeNumberPlateTag = createStyledTag(`${name}NumberPlate`, {
   height: '100%',
   display: 'block',
-  width: '3rem',
+  width: '30px',
```
Comment (ej9x) [inline src/components/Code/Code.theme.js]: `30px`

File: src/components/Checkbox/Checkbox.test.js (hunk 2 above)
Comment (zouxuoz) [reply src/components/Checkbox/Checkbox.test.js]: @ej9x fixed

File: src/components/TextArea/TextArea.js (hunk 3 above)
Comment (zouxuoz) [reply src/components/TextArea/TextArea.js]: @ej9x fixed
